pkg/operator/staticresourcecontroller/static_resource_controller.go

```diff
@@ -88,6 +88,10 @@ type conditionalManifests struct {
 
 	manifests resourceapply.AssetFunc
 
+	// manifestObjModifier is called on manifest resources. Can be used, e.g. for setting a namespace or an owner reference.
+	// The function calls should not result in a different name/namespace on successive calls for the same object.
```

Comment:
I'm not sure I fully understand this. We are going to call this to modify namespace, so function calls will result in a different name/namespace?

Reply:
I mean, you can change the stored asset to your namespace, but if you change it to other namespace during the runtime of the operator, it will not achieve the resource migration from one ns to another.

Reply:
added to the comments